packages/app-store/routing-forms/components/_components/ResultSection.tsx

```diff
@@ -0,0 +1,308 @@
+                      Attribute logic matched
+                    </span>
+                  </div>
+                  <Badge
+                    data-testid="attribute-logic-matched"
+                    variant={membersMatchResult.checkedFallback ? "error" : "success"}>
+                    {membersMatchResult.checkedFallback ? "No" : "Yes"}
+                  </Badge>
+                </div>
+                <div className="flex items-center justify-between">
+                  <div className="flex items-center gap-1">
+                    <div className="border-subtle bg-default z-10 rotate-180 rounded-lg border p-1">
+                      <Icon name="split" className="h-4 w-4" />
+                    </div>
+                    <span className="text-emphasis text-sm font-medium leading-none">Attribute fallback</span>
+                  </div>
+                  <Badge
+                    data-testid="attribute-logic-fallback-matched"
+                    variant={membersMatchResult.checkedFallback ? "success" : "gray"}>
+                    {membersMatchResult.checkedFallback ? "Yes" : "Not needed"}
+                  </Badge>
+                </div>
+                <div className="flex items-center justify-between">
+                  <div className="flex items-center gap-1">
+                    <div className="border-subtle bg-default z-10 rounded-lg border p-1">
+                      <Icon name="user" className="h-4 w-4" />
+                    </div>
+                    <span className="text-emphasis text-sm font-medium leading-none">Contact owner</span>
+                  </div>
+                  <Badge variant={membersMatchResult.contactOwnerEmail ? "success" : "gray"}>
+                    {membersMatchResult.contactOwnerEmail || "Not found"}
+                  </Badge>
+                </div>
+              </div>
+            </ResultsSection>
+
+            {membersMatchResult.teamMembersMatchingAttributeLogic && (
+              <ResultsSection
+                title={`${membersMatchResult.teamMembersMatchingAttributeLogic.length} next in queue`}
+                icon="user-check"
+                hint={
+                  <div className="flex items-center gap-2 px-2 py-1">
+                    <Icon name="info" className="h-3 w-3" />
+                    <span data-testid="matching-members" className="text-subtle text-sm">
+                      {t("routing_preview_more_info_found_insights")}
+                    </span>
+                  </div>
+                }>
+                <div className="divide-subtle divide-y">
+                  {membersMatchResult.teamMembersMatchingAttributeLogic.map((member, index) => (
+                    <TeamMember
+                      key={member.id}
+                      email={member.email}
+                      name={member.name}
+                      score={membersMatchResult.perUserData?.weights?.[member.id] || 0}
```

Comment:
```suggestion
                      score={membersMatchResult.perUserData?.bookingShortfalls?.[member.id] || 0}
```

Should be the booking shortfall not the weight. I also added a comment in figma, we not to make this clear what this number means